IAM/Frequently_asked_questions.mediawiki

```diff
@@ -211,11 +211,58 @@ If you own a volunteer LDAP account, then you can upgrade your mozillians profil
+
+When you login using an email link, that link is valid for 15 minutes from when you request it.
+This expiration window of 15 minutes is driven both by security considerations and a desire for
+a positive user experience. The link is short lived so that there is a limited window
+of time during which a potential attacker could use the link if they were able to get access
+to it. This is especially important due to the inherently insecure nature of email
+transmission. 
+
+During the past 18+ months, experience shows that for a vast majority of users, this 15 minute expiration window has no effect on
+them as they receive the email link in their inbox mere seconds after they click the
+button requesting the link. Some users however do not receive the email login link
+immediately.
+
+Short delays in delivery are just part of how email delivery works.
+Longer delays however can be caused by a feature called [https://en.wikipedia.org/wiki/Greylisting Greylisting]. Continue on to the question below for further information on Greylisting.
+
+==== '''Q''': ''What is email Greylisting?'' ====
+Some email providers institute Greylisting on all inbound email for their users
+as a measure to reduce spam. Greylisting temporarily rejects email from mail servers
+where the sender hasn't communicated with the recipient before or due to some other
+signal indicating the email may be spam. A mail server that is Greylisting expects
+that a valid sender and mail server will continue to retry sending the email over
+time and the Greylisting mail server will eventually accept the mail. In the case
+of a "transactional" email like the email login link,
+[https://en.wikipedia.org/wiki/Greylisting#Delayed_delivery_issues Greylisting prevents users from being able to do a real-time login].
+
+ ==== '''Q''': ''What can I do about my email provider's Greylisting?'' ==== 
+Users who have email providers that use Greylisting will likely see this type of
+severely delayed transactional emails from other senders as well, for example
+when they sign up to a new web site and that web site sends them an email with a
+link to confirm that the user controls the email address they signed up with.
+
+Unfortunately, in order for users with email providers that utilize Greylisting
+to work around this problem, they may need to contact their email provider or
+look in their providers documentation to see if there are whitelisting options
+available to them. There's nothing that Mozilla can do on the sending side to
+force the user's Greylisting mail server to accept and deliver the email.
+
+Users sometimes ask to just increase the expiration time, for example to 30 or 60
+minutes. We deliberately have not made this change because doing so would decrease our systems' security while not addressing the root cause of this problem (Greylisting).
+Instead, we ask that users either:
+* contact their email provider to ask to have Greylisting disabled
+* have Mozilla whitelisted
```

Comment:
On line 262, it suggests to a user to have Mozilla whitelisted. I would suggest that this be expanded upon with specific information about about domains need to be whitelisted, or a link to a separate FAQ item along the lines of "how do I get Mozilla whitelisted with my e-mail provider", which lists specific domains that need to be whitelisted.

Reply:
Ya, I mean this (whitelisted) in the most abstract sense since I've never used a Greylisting email provider, run a greylisting MTA or know what their capabilities are. My hope is that if a user contacts their email provider, explain the problem and use the world whitelist, the email provider may know how to do that and instruct them what to do and what to tell them. I don't know if this is something like whitelisting AWS SES server IPs or whitelisting sender domains or something else. We could optimistically add the sender email domain here in hopes that that's what the provider would need.